mobile/android/android-application-manager.ts

```diff
@@ -83,6 +84,25 @@ export class AndroidApplicationManager extends ApplicationManagerBase {
 		return this.$androidProcessService.getDebuggableApps(this.identifier);
 	}
 
+	public getDebuggableAppViews(appIdentifiers: string[]): IFuture<IDictionary<Mobile.IDebugWebViewInfo[]>> {
+		return ((): IDictionary<Mobile.IDebugWebViewInfo[]> => {
+			let mappedAppIdentifierPorts = this.$androidProcessService.getMappedAbstractToTcpPorts(this.identifier, appIdentifiers).wait(),
+				applicationViews: IDictionary<Mobile.IDebugWebViewInfo[]> = {};
+
+			_.each(mappedAppIdentifierPorts, (port: number, appIdentifier: string) => {
+				if (port) {
+					let apps = this.$httpClient.httpRequest(`http://127.0.0.1:${port}/json`).wait().body;
+					applicationViews[appIdentifier] = JSON.parse(apps);
```

Comment:
We could do with some error prevention/handling hare - if `apps` is false-like `JSON.parse` will throw an error